### ISSUE TITLE
Add sendText and operator << to Player and Playerset

### ISSUE THREAD

**Issue by heinermann** [bwapi/bwapi#485]
_From [AHeinerm](https://code.google.com/u/AHeinerm/) on May 20, 2013 01:00:25_

The addition of sendText as a member of Player will allow the client to easily send private messages to specific players in the game (a lacking feature).

Similarly, this addition would also be made to Playerset so that messages can be sent to player groups.

This will allow the use of stuff like: `Broodwar->allies().sendText("Hey!")`, instead of using the unclear `sendTextEx`. It will also deprecate `sendTextEx`.

Then, we can put Player in a wrapper for overloading operator << like in Broodwar. The same operator << can easily be put into Playerset.

Code then becomes:

```
Broodwar->allies() << "Hey!" << endl;
someUnit->getPlayer() << "Your " << someUnit->getType() << " is in my way." << endl;
```

---

tl;dr
- Add sendText member to Player and Playerset.
- Add operator << overload to Playerset.
- Wrap Player to overload operator <<.
- Deprecate/repurpose sendTextEx.

_Original issue: http://code.google.com/p/bwapi/issues/detail?id=488_
